clientapp/app/page.tsx

```diff
@@ -1,48 +1,42 @@
 import BodyTypePage from './components/BodyTypePage/BodyTypePage'
-import HeightPage from './components/EnterHeightPage/HeightPage'
-import WeightPage from './components/EnterWeightPage/WeightPage'
+import EnterMeasurements from './components/EnterMeasurementsPage/EnterMeasurements'
 import GeneratePlan from './components/GeneratedPlanPage/GeneratePlan'
 import LandingPage from './components/LandingPage/LandingPage'
+import { MeasurementsProvider } from './components/shared/context/MeasurementsData'
 
 export default function Home() {
   return (
-    <main className='overflow-x-hidden'>
-      <div className='w-screen h-screen' style={{
-        backgroundImage:`url('/ForkLandingPage.jpg')`,
-        backgroundSize:'cover',
-        backgroundPosition:'center'
-      }}>
-        <LandingPage/>
-      </div>
+    <MeasurementsProvider>
+      <main className='overflow-x-hidden'>
+        <div className='w-screen h-screen' style={{
```

Comment:
I'm noticing that this forkLandingPage div appears 3 times in this file. Any way we can avoid this?